docs/content/docs/_index.md

```diff
@@ -333,15 +333,16 @@ You can check whether a left side is contained in a right side using the `in` op
 ```
 
 Only literals/variables resulting in an array, a string and an object are supported in the right hand side: everything else
-will raise an error.
+will raise an error. While in the left hand side only literals/variables resulting in a number, a string and a boolean are supported.
 
 
 ## Manipulating data
 
 ### Assignments
 You can assign values to variables during the rendering.
 Assignments in for loops and macros are scoped to their context but
-assignments outside of those will be set in the global context.
+assignments outside of those will be set in the global context. Furthermore, assignments
+in for loop are valid until the end of the iteration only.
```

Comment:
of the current iteration

Otherwise it sounds like it's valid during the whole for loop.